### DISCUSSION TITLE
How to use send_command from home assistant ?

### DISCUSSION THREAD

**Discussion by francisp2** [timmo001/system-bridge#727] [Help] [answered]
![image](https://user-images.githubusercontent.com/52328611/129773770-bc4b3cd1-4b08-401f-ad26-9945e0a4adad.png)

I tried all kind of variations, 
  command: "c:\\windows\\System32\\shutdown.exe"
  command: c:\\windows\\System32\\shutdown.exe
  command: 'c:\\windows\\System32\\shutdown.exe'
  command: "%windir%\\System32\\shutdown.exe"
  command: %windir%\\System32\\shutdown.exe
  command: '%windir%\\System32\\shutdown.exe'
  command: "c\windows\System32\shutdown.exe"
  command: c\windows\System32\shutdown.exe
  command: 'c:\windows\System32\shutdown.exe'
  command: "%windir%\System32\shutdown.exe"
  command: %windir%\System32\shutdown.exe
  command: '%windir%\System32\shutdown.exe'
  command: shutdown

Nothing works

**Comment by francisp2** [comment, marked as answer]
Stupid me

![image](https://user-images.githubusercontent.com/52328611/129843067-8aad2b4c-6c1a-4151-9bec-309da96d63b9.png)

**Comment by Iceman248** [comment]
I don't even have the "Send Command" as a service available through HA. I use Linux. Does that only work on Windows?
